config.go

```diff
@@ -345,6 +347,11 @@ func loadAndValidateConfig(interceptor signal.Interceptor) (*Config, error) {
 	// Show the version and exit if the version flag was specified.
 	appName := filepath.Base(os.Args[0])
 	appName = strings.TrimSuffix(appName, filepath.Ext(appName))
+	if preCfg.ShowVersion {
+		fmt.Println(appName, "version", Version(),
+			"commit="+Commit)
```

Comment:
The `Version()` function already returns `commit=...`.
This currently prints:
```
litd version 0.13.3-alpha commit=v0.13.3-alpha-12-gbaebcbdc4b86cb9453445102295cea8a030cdd57 commit=v0.13.3-alpha-12-gbaebcbdc4b86cb9453445102295cea8a030cdd57
```

Reply:
ah true - thanks!